gemfiles/rails51.gemfile

```diff
@@ -13,6 +13,7 @@ gem "unicorn"
 gem "actionpack", "~> 5.1.0"
 gem "actionview", "~> 5.1.0"
 gem "activerecord", "~> 5.1.0"
+gem "rspec-rails", group: [:development, :test]
```

Comment:
Why did this move here? Could it not have stayed in the `:development, :test` group below?

Reply:
These gemfiles are generated by Appraisal. I needed to set up the appraisals to use different versions of rspec-rails, and the changes in this PR were the only way I could think to do it. I’ll double check whether there is another way to do this that leaves these gemfiles as they were.

Reply:
Ah fair enough. I don't think it's a problem. I was mostly curious.

Reply:
Using 

```
  group :development, :test do
    gem "rspec-rails"
  end
```

in the Appraisal file seems to have done the correct thing.